pkg/config/datastore.go

```diff
@@ -37,6 +37,9 @@ type Data struct {
 	// By default, Porter requires the --force flag to be specified to overwrite a bundle or image.
 	ForceOverwrite bool `mapstructure:"force-overwrite"`
 
+	// Force skip rebuild invocation image even if out-of-date detected
+	AutoBuildDisabled bool `mapstructure:"autobuild-disabled"`
```

Comment:
Is this necessary for this PR?

Reply:
No. It came out in the process of resolving conflicts. Should I try to delete?